x-pack/plugins/security_solution/public/overview/components/host_overview/endpoint_overview/translations.ts

```diff
@@ -10,7 +10,7 @@ import { i18n } from '@kbn/i18n';
 export const ENDPOINT_POLICY = i18n.translate(
   'xpack.securitySolution.host.details.endpoint.endpointPolicy',
   {
-    defaultMessage: 'Integration',
```

Comment:
Do you need to remove the existing translations to make sure they're updated?

Reply:
Because the 'id' did not change, I don't think so. Its only when the id changes or removed that you likely need to clean up the translations